src/utils/device.js

```diff
@@ -33,7 +33,9 @@ if (isWebXRAvailable) {
       return;
     }
     if (sceneEl.hasLoaded) {
-      sceneEl.components['vr-mode-ui'].updateEnterInterfaces();
+      if (sceneEl.components['vr-mode-ui']) {
+        sceneEl.components['vr-mode-ui'].updateEnterInterfaces();
```

Comment:
(optional ) I think you can also do 

      sceneEl.components['vr-mode-ui']?.updateEnterInterfaces();

Reply:
Yeah you can but I'm getting this in my presubmit hook:
```
/Users/parismorgan/repo/8frame/src/utils/device.js
  36:40  error  Parsing error: Unexpected token   null

✖ 1 problem

husky - pre-commit hook failed (add --no-verify to bypass)
```
which I think is from it being on an old node version: https://stackoverflow.com/questions/59574047/how-to-use-optional-chaining-in-node-js-12

So just going to leave, thanks though, good call.